README.md

```diff
@@ -95,6 +95,53 @@ $ curl -XGET http://ip:port/users
 
 The pivot used for the synchronization in LSC connector is email address. For this case, `james-user@james.org` is stored in `email` attribute.
 
+### Domain contact Synchronization
+
+For example, it can be used to synchronize the domain contact stored in the LDAP of an OBM instance to the TMail Server(s) of a TMail deployment.
```

Comment:
```suggestion
For example, it can be used to synchronize the domain contact stored in a LDAP instance to the TMail Server(s) of a TMail deployment in order to enpower auto-complete.
```